ambry-store/src/test/java/com.github.ambry.store/CuratedLogIndexState.java

```diff
@@ -371,6 +385,21 @@ MockId getIdToDeleteFromLogSegment(LogSegment segment) {
   }
 
   /**
+   * Gets the expected size of the valid data at {@code referenceTimeMs} in {@code segment}.
+   * @param segment the {@link LogSegment} whose valid size is required.
```

Comment:
Not the author of this but i think valid data size might be a misleading name for valid size per log segment. Since delete records are considered valid here and delete records doesn't really belong to the **data** category.